docs/source/advanced/no-runtime.md

```diff
@@ -10,13 +10,15 @@ For this, remove the `com.apollographql.apollo:apollo-runtime`dependency and rep
   implementation("com.apollographql.apollo:apollo-api:x.y.z")
 ```
 
+## Parsing HTTP body
+
 All `Operation` instances provide an API to parse `Response` from raw `okio.BufferedSource` source that represents http response body returned by the GraphQL server.
 If for some reason you want to use your own network layer and don't want to use fully featured `ApolloClient` provided by `apollo-runtime` you can use this API:
 
 ```java
     okhttp3.Response httpResponse = ...;
 
-    Response<Query.Data> response = new Query().parse(httpResponse.body().source());
+    Response<Operation.Data> response = new Query().parse(httpResponse.body().source());
```

Comment:
I thought operation here would be more correct. `Query` was referring to a generated type, right?

Reply:
I think `Query` was referring to https://github.com/apollographql/apollo-android/blob/65c4fff8b98f9c8f5949adb6a3b2e694489dfee0/apollo-api/src/main/java/com/apollographql/apollo/api/Query.java.

But this all should work with Mutation as well so using `Operation.Data` seems better 👍